config/webpack.config.js

```diff
@@ -87,7 +88,8 @@ if (__DEV__) {
   debug('Enabling plugins for live development (HMR, NoErrors).')
   webpackConfig.plugins.push(
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin()
+    new webpack.NoErrorsPlugin(),
+    new OpenBrowserPlugin({ url: project.compiler_public_path })
```

Comment:
Using `project.compiler_public_path` just creates a url of `/`, which won't open in a browser. You need to have `http` or `https` in front of the url  for it to open in a browser, and you need to pass in the localhost address.
Try this instead: `url: http://localhost:${project.server_port}`